documentation/pom.xml

```diff
@@ -181,6 +181,9 @@
     <profiles>
         <profile>
             <id>documentation-pdf</id>
+            <activation>
+                <activeByDefault>!skipPdfDocumentation</activeByDefault>
```

Comment:
oops yes I realized that when testing as I initially used just `activeByDefault`. In fact what I committed later is illegal, activeByDefault doesn't expect a property..